src/components/Header.vue

```diff
@@ -0,0 +1,22 @@
+<template lang="pug">
+    b-jumbotron#header.text-center(
+        bg-variant="info"
+        text-variant="light"
+    )
+        h1(@click="goHome") SAoT
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+export default class Header extends Vue {
+    goHome() {top.location.href = "/";};
```

Comment:
Use
```javascript
this.$router.push("/");
```
instead of reloading page. See: https://router.vuejs.org/guide/essentials/navigation.html